recipes/extensisq/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "extensisq" %}
+{% set version = "0.0.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/extensisq-{{ version }}.tar.gz
+  sha256: e99ff36797d62f8316c76a1230a49d3a9738cb19b276451fa414d4bd727b7d48
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numpy >=1.4.0
+    - python >=3.6
+    - scipy >=1.0.0
+
+test:
+  imports:
+    - extensisq
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/WRKampi/extensisq
+  summary: extend solve_ivp of scipy.integrate with OdeSolver objects
+  license: MIT
+  license_file: LICENSE
```

Comment:
Add a comment:

```yaml
# License file manually packaged. See: https://github.com/WRKampi/extensisq/pull/1
license_file: LICENSE
...
```

I've opened a PR upstream (https://github.com/WRKampi/extensisq/pull/1) to fix having to manually package the license file. In a future release if this PR is merged or if the license file is included in some other way, this comment is a visual reminder to check, and if the license file is in the source distribution, you can remove the comment and delete the manually packaged copy of the license.

Reply:
I have made the modifications. The packaged file now contains the licence. I created a new version for this. Is this the only way to do it? Or did I make it more difficult that I needed to? It is my first contribution to conda forge.